cdhweb/pages/templates/cdhpages/blocks/accordion_block.html

```diff
@@ -2,8 +2,9 @@
 
 <div data-component="accordion" class="sk-accordion">
   {% for item in self.accordion_items %}
+    <h2 class="sr-only">{{ item.heading }}</h2>
     <details>
-      <summary>
+      <summary aria-label="Toggle section: {{ item.heading }}">
```

Comment:
I realised the accordions didn't have any H2s. This is expected, since the rich-text content blocks within only allow h3+.

As for the `sr-only` and `aria-label`, that was the upshot of [this thread](https://springload.slack.com/archives/C027E7YDF5Z/p1694123285126019?thread_ts=1693366675.860289&cid=C027E7YDF5Z).